### ISSUE TITLE
Question/Feature: Is it possible to get HTML using manet?

### ISSUE THREAD

**Issue by tadas-subonis** [vbauer/manet#58]
I would like to use manet through its RESTful API to get HTML content of the page.

Is it possible?

**Comment by vbauer** [comment]
@tadas-subonis Hello, could you please provide some example? What is the difference between this and simple GET request to some URL?

**Comment by tadas-subonis** [comment]
If I do:
```
curl http://localhost:8891/?url=github.com > github.png
```

but I want to do:

```
curl http://localhost:8891/?url=github.com&format=html > page_content.html
```

so I would get the loaded page's HTML.

**Comment by vbauer** [comment]
Do you mean that `curl http://github.com > page_content.html` is not enough, because some of your logic on client side could modify DOM?

**Comment by tadas-subonis** [comment]
yeah. Like most of the pages built with angular - they load their content
with JavaScript.

On Sun, May 1, 2016 at 4:50 PM, Vladislav Bauer <notifications@github.com>
wrote:

> Do you mean that curl http://github.com > page_content.html is not
> enough, because some of your logic on client side could modify DOM?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/vbauer/manet/issues/58#issuecomment-216043161>
>



-- 

Kind Regards,
Tadas Šubonis

**Comment by vbauer** [comment]
I've just added "html" format to fetch HTML data. Changes are available only in master at the current moment. Could you please check this features?

**Comment by tadas-subonis** [comment]
Works great :+1: 

When are you planning to release it? :)

**Comment by vbauer** [comment]
Thank you for testing! It is already available in 0.4.15.
